gapis/service/service.proto

```diff
@@ -1251,3 +1255,16 @@ message GetTimestampsResponse {
     Error error = 2;
   }
 }
+
+// Starts a stream of commands to be executed
+message StreamStartRequest {
```

Comment:
Extra stuff?

Reply:
Thanks . I thought I had cleaned them all up, but obviously not quite.